packages/ai-jsx/package.json

```diff
@@ -303,7 +309,6 @@
     "graphql": "^16.6.0",
     "ink": "^4.2.0",
     "ink-syntax-highlight": "^1.0.1",
-    "jest": "^29.5.0",
```

Comment:
lol, it would be been embarrassing to have this in the launch. Thanks for finding it.